archetype-java-builders/src/main/resources/archetype-resources/src/main/java/ExampleJLoadScenarioProvider.java

```diff
@@ -16,10 +16,18 @@
 import com.griddynamics.jagger.user.test.configurations.termination.JTerminationCriteriaIterations;
 import com.griddynamics.jagger.user.test.configurations.termination.auxiliary.IterationsNumber;
 import com.griddynamics.jagger.user.test.configurations.termination.auxiliary.MaxDurationInSeconds;
+import com.griddynamics.jagger.util.JaggerPropertiesProvider;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 
-public class ExampleJLoadScenarioProvider {
-    
-    public static JLoadScenario getExampleJaggerLoadScenario() {
+@Configuration
+public class ExampleJLoadScenarioProvider extends JaggerPropertiesProvider {
+
+    @Bean
+    public JLoadScenario exampleJaggerLoadScenario() {
+
+        // Example of using JaggerPropertiesProvider
```

Comment:
please make some example of customer property. e.g. to parametrize test duration and/or load value

benefit of such parametrization, you can restart load test project without recompiling, just with different values for parameters (set via system properties)